lib/image/options/write.ex

```diff
@@ -240,10 +256,10 @@ defmodule Image.Options.Write do
   end
 
   defp image_type_from("", suffix) do
-    {:ok, suffix}
+    {:ok, String.trim_leading(suffix, ".")}
```

Comment:
This allows suffixes with or without a leading dot.
So both `suffix: ".jpg"` and `suffix: "jpg"` will work.